### ISSUE TITLE
Dimensionality reduction 

### ISSUE THREAD

**Issue by mattskier** [mattskier/ravens-dse-511#15]

**Comment by prachi1294** [comment]
Thank you Matt. Resumed working on it now but will continuing from tomorrow
onwards. Good luck for test!

**Comment by mattskier** [comment]
looks good with the PCA, with the RF classifiers, I'm guessing that increasing the depth may help the fact that we're getting everything in categories A/B

**Comment by prachi1294** [comment]
yes, I have increased the depth, and accuracy got increased magically. I have committed the latest changes in GitHub with new accuracy.

**Comment by prachi1294** [comment]
But increasing depth didn't change the confusing matrix, it still falling in A/B categories. So far we have gotten the best accuracy so nevermind.
